src/status_im2/contexts/syncing/syncing_devices_list/view.cljs

```diff
@@ -21,21 +20,45 @@
 
 (defn view
   []
-  [rn/view {:style style/container-main}
-   [navigation-bar]
-   [rn/view {:style style/page-container}
-    [rn/view {:style style/title-container}
-     [quo/text
-      {:size   :heading-1
-       :weight :semi-bold
-       :style  {:color colors/white}} (i18n/label :t/syncing)]
-     [quo/button
-      {:size     32
-       :icon     true
-       :on-press #(rf/dispatch [:navigate-to :settings-setup-syncing])}
-      :i/add]]
-    [rn/view {:style style/devices-container}
-     [device/view
-      {:name         "iPhone 11"
-       :this-device? true
-       :device-type  :mobile}]]]])
+  (let [devices           (rf/sub [:pairing/installations])
+        user-device       (first devices)
+        other-devices     (rest devices)
+        paired-devices    (filter :enabled? other-devices)
+        un-paired-devices (filter #(not (:enabled? %)) other-devices)]
```

Comment:
For `unpaired-devices` you could use `remove` instead.

But if you wish, you could use `group-by` to get both in a single pass, followed by a destructuring, like:

```clojure
{:keys [paired unpaired]} (group-by #(if (:enabled? %) :paired :unpaired)) other-devices)
```

wdyt?

Reply:
yep, will adjust and also the same for your other comments. Thanks for the review!